### PR TITLE
Add CI Pipeline + More

### PR DESCRIPTION
**Pull request by inverse**
- Add CI pipeline to run tests
- Move tests out to top level dir
- Refactor to use module loading 

To launch run `python -m algobot`

Please test the changes locally... It wasn't running for me before due to the path references to the UI - I also didn't apply the module fixes to all code, only the ones it was complaining about.

### CODE REVIEW

**Comment by ZENALC** [comment]
Very nice PR!

File: requirements-test.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+-r requirements.txt
```
Comment (ZENALC) [inline requirements-test.txt:1]: I didn't know we could have a seperate requirements.txt for tests. Thanks a lot for sharing!

**Review by ZENALC — APPROVED** [review]

File: requirements-test.txt (hunk 1 above)
Comment (inverse) [reply requirements-test.txt:1]: Yeah :)

There are even alternative ways to manage dependencies.

- [pipenv](https://pipenv.pypa.io/en/latest/)
- [poetry](https://python-poetry.org/)

**Comment by inverse** [comment]
@ZENALC one thing - you might want to squash the pull request before merging in order to get clean history and make it easier to revert when things don't work out.

See: https://medium.com/@shubhpaliwal98/when-to-use-squash-merge-and-rebase-43da3571dcbe

Also you can set this up as the default button action in the Github settings of the repository.

image


Unchecking this does it 👍🏼

**Comment by ZENALC** [comment]
@inverse Thank you! I have updated the settings.

**Pull request merged**